spec/services/moment_keywords_spec.rb

```diff
@@ -0,0 +1,34 @@
+describe MomentKeywords do
+  subject(:moment) {FactoryBot.build(:moment,
+                                     categories: [build(:category, name: 'free', description: 'Description')],
+                                     moods: [build(:mood, name: 'Name', description: 'Blog^^')],
+                                     strategies: [build(:strategy, name: 'Name', description: 'books@!##.')],
+                                     name: "ADDICTION",
+                                     why: "self-care.",
+                                     fix: "@Teachers!!" )}
+
+  let(:keywords) { MomentKeywords.new(moment).extract_moment_keywords }
```

Comment:
It it refers to the class being testes (`MomentKeywords`), this needs to be the subject

```suggestion
  subject(:keywords) { MomentKeywords.new(moment).extract_moment_keywords }
```